fattmerchantorg/fattmerchant_python_client
# [SPRINT-106][FAT-4258] team/registration PUT functionality

## **[Ticket FAT-4258](https://fattmerchant.atlassian.net/browse/FAT-4258)**

> As a python developer, I want to update merchants registration.

## What did I do?
* Created TeamRegistration model
* Added new `update_registration` function to  TeamController
* Added testing scenario to test.py 

## Testing
- Start venv (see README on this repo on how to setup environment)
- Modify test.py : 
  - Get a jwt from your local api and replace at the top of the file
  - uncomment lines  32 -35
  - run `python test.py` from your venv 

## PR notes contain:

* [X] Checklist for any deployment dependencies (such as new env vars, back-end dependencies, etc)

## PR Checklist:

* [X] **I tested this**
* [X] I updated from the base branch
* [X] I used the following PR title format: "[SPRINT-XX][fat-xx] BLAH"
* [X] Add "[EFS]" to the PR title if it was in the ticket title, e.g. "[SPRINT-XX][fat-xx][EFS]"
* [X] I looked over the Files Changed tab
* [X] I consulted with the ticket's product owner (if necessary)
* [X] I requested a peer review
